prototypes/recordedfuture.yml

```diff
@@ -4,11 +4,10 @@ description: >
     enables organizations to defend themselves at the speed and scale of the
     Internet. Our service continuously analyzes content from the entire Web
     to give you unmatched insight into emerging threats.
-    Follow us on Twitter @RecordedFuture
 
 prototypes:
     IPRiskList:
-        author: MineMeld Core Team
+        author: MineMeld Core Team, Recorded Future Inc.
```

Comment:
I would not change the original author of this prototype, but I would mark this as deprecated. The new Master Miner can do more than the original miner.

Reply:
I will change the name back to its original name. For marking it as deprecated, Should I write something like this :

prototypes:
    IPRiskList:
        author: MineMeld Core Team (deprecated)

We are planning to keep this miner active for our clients.

Reply:
you can change `development_status` to `DEPRECATED`

Reply:
and leave the author as it is